.github/workflows/post-release-action.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+
+# This Github action updates the target-branch references for dependabot and other actions
+name: Post Release Action
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_name:
+        description: 'Name of the new release branch, i.e. release-v1.0.0'
+        required: true
+
+jobs:
+  update-references:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Update branch references
+        run: |
+          BRANCH_NAME=${{ github.event.inputs.branch_name }}
+          grep -rl 'target-branch:' . | xargs sed -i "s/target-branch: \".*\"/target-branch: \"${BRANCH_NAME}\"/g"
+
+      # Needed for signing commits using Github App tokens
+      # See: https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#commit-signing
+      - uses: actions/create-github-app-token@v1.11.0
+        id: generate-token
+        with:
+          app-id: ${{ vars.CSM_RELEASE_APP_ID }}
+          private-key: ${{ secrets.CSM_RELEASE_APP_PRIVATE_KEY }}
+
+      - name: Create Pull Request
```

Comment:
This action is titled 'Create Pull Request', is it just a general action to create a pull request or is it specific to the release branch? If it's a general action, does it have any inputs that can be used to create PRs for specific needs?

Reply:
The PR is specific to the file changes that come from the step `Update branch references`

Reply:
Should we rename the action to something that's more specific, then? 'Create PR for Release Updates' or similar?

Reply:
Fixed